lib/models/migration.js

```diff
@@ -45,6 +45,47 @@ module.exports = function(Migration, options) {
   };
 
   /**
+   * Remote Method: Run specific migration by name.
+   *
+   * @param {String} [name] Name of migration script to run.
+   * @param {String} [record] Record the migration runtime to database.
+   * @param {Function} [cb] Callback function.
+   */
+  Migration.migrateByName = function(name, record, cb) {
+    name = name || '';
+    record = record || false;
+    cb = cb || utils.createPromiseCallback();
+    assert(typeof name === 'string', 'The to argument must be a string, not ' + typeof name);
+    assert(typeof cb === 'function', 'The cb argument must be a function, not ' + typeof cb);
+
+    Migration.log.info(name, 'running.');
+    const scriptPath = path.resolve(path.join(Migration.migrationsDir, name));
+
+    try {
+      require(scriptPath).up(Migration.app, function(err) {
+        if (err) {
+          Migration.log.error(name, 'error:');
+          Migration.log.error(err.stack);
```

Comment:
I'd suggest combining these 2 lines so that only one error message is logged if there is an error. Perhaps make the error message a little more descriptive too:

```
Migration.log.error(`Error running migration script ${name}:`, err);
```